addons/account/tests/test_fiscal_position.py

```diff
@@ -268,3 +269,22 @@ def test_30_fp_delivery_address(self):
             self.env['account.fiscal.position'].get_fiscal_position(partner_us_no_vat.id, partner_us_no_vat.id),
             fp_eu_extra
         )
+
+    def test_fiscal_position_constraint(self):
+        """
+        Test fiscal position constraint by updating the record
+        with only zip_from value
+        """
+        fiscal_position = self.fp.create({
+            'name': 'Test fiscal',
+            'auto_apply': True,
+            'country_id': self.be.id,
+            'vat_required': True,
+            'sequence': 10,
+        })
+        with self.assertRaises(ValidationError):
```

Comment:
You can add the two other scenarios (`to` bound only -> fail; two bounds -> success)

Reply:
Hello @yosa-odoo 

Requested changes are done.

Thank You :slightly_smiling_face: